changelogs/fragments/6785-openbsd_pkg_pkg_info_handling.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - openbsd_pkg - The pkg_info(1) behavior has changed in OpenBSD >7.3. The error message ``Can't find`` should not lead to an error case (https://github.com/ansible-collections/community.general/pull/6785).
```

Sorry, I also missed a detail while reviewing (damn early mornings ;) )
```suggestion
  - openbsd_pkg - the pkg_info(1) behavior has changed in OpenBSD >7.3. The error message ``Can't find`` should not lead to an error case (https://github.com/ansible-collections/community.general/pull/6785).
```